docs/content-guidelines.md

```diff
@@ -0,0 +1,198 @@
+
+This type of metadata determines where in the navigation structure of the Luigi website the document will be shown. It must be written as a json object surrounded by `<!--meta` `meta-->` tags. For example:
+
+```json
+<!-- meta
+{
+  "node": {
+    "label": "Overview",
+    "category": {
+      "label": "Basics"
+    },
+    "metaData": {
+      "categoryPosition": 1,
+      "position": 0
+    }
+  }
+}
+meta -->
+```
+
+### Alert blocks metadata
+
+To draw the reader's attention to something, you can use the quote block option in Markdown. Directly above the quote block, you need to add metadata which determines whether the alert box on the website will be green (`success`) or yellow (`warning`).
+
+Use one of these three options for alert blocks:
+
+```
+<!-- add-attribute:class:warning -->
+>**NOTE:** Necessary information
+```
+```
+<!-- add-attribute:class:success -->
+>**TIP:** Useful, but not strictly necessary information
+```
+```
+<!-- add-attribute:class:warning -->
+>**WARNING:** Very important information
+```
+
+
+
+## Audience and language
+
+The audience of this documentation consists mainly of developers interested in implementing a micro frontend UI solution. It is assumed the reader already has basic knowledge of web development. Do not explain general concepts unrelated to Luigi except if they are instrumental for working with the feature you're describing.
+
+When writing documentation, you should adhere to a few basic rules:
+- Use active voice. For example, instead of writing "Luigi Client should be installed...", write "Install Luigi Client..."
+- Do not use slang or abbreviations. This also means you should not use contractions ("don't" instead of "do not") or short forms ("info" instead of "information").
+- Use the present tense.
+- Use concise language and avoid long blocks of text. Lists, tables, or subheadings can help you with that.
+- Give practical examples of features instead of only using words to explain them. Additionally, link to the [Luigi Fiddle](https://fiddle.luigi-project.io/) as a tool where users can experiment with features.
+
+## Format
+
+This section provides you with guidelines on how to format and organize your text.
+
+### Headings
+
+Use H1 headings (preceded by # in Markdown) only at the start of the document to indicate the document name.
+
+Only use H2 and H3 headings to organize your content into categories. Do not use H4 (####) headings or lower.
+
+### Lists
+
+Lists are very useful for breaking up text and providing instructions.
+
+- Use bullet points (created with * or -) for lists involving general explanations.
+- Use numbered lists only for step-by-step instructions.
+
+### Tables
+
+Use tables when content needs comparison or to organize small bits of information. Keep in mind that in Markdown it is not possible to break up the content of a table cell into separate lines or bullet points, therefore tables are not useful when they contain long sentences or lists.
```

Comment:
```suggestion
Use tables when content needs comparison or to organize small bits of information. Keep in mind that in Markdown it is not possible to break up the content of a table cell into separate lines or bullet points, therefore tables are not useful when they contain long sentences or lists.
```
You actually can break it in separate lines using `<br>`  or some other tricks: https://gist.github.com/shaunlebron/746476e6e7a4d698b373 :D I'm not saying we necessary advise that, but it is possible and sometimes I do use it (when I need to list more terms). 
But the general comment that you shouldn't add long lines is good :)